package.json

```diff
@@ -1,50 +1,45 @@
 {
   "name": "react-page-visibility",
-  "version": "0.0.2",
+  "version": "1.0.0",
   "description": "Declarative, nested, stateful, isomorphic page visibility for React",
   "author": "Gilad Peleg <giladp007@gmail.com> (http://giladpeleg.com)",
   "repository": "pgilad/react-page-visibility",
-  "keywords": "react, components, visibility, page, higher-order, browser",
-  "main": "index.js",
+  "keywords": "react, components, visibility, page, higher-order, browser, react-component",
+  "main": "dist/index.js",
+  "module": "src/index.js",
   "files": [
-    "index.js",
-    "visibility.js"
+    "dist",
```

Comment:
This files property caught me out when publishing to npm, even if you have a `.npmignore` file defined it gets ignored and this takes precedence with no warning from npm. At least I learn't something new 😄